fastlane/lanes/release.rb

```diff
@@ -12,134 +12,134 @@
+    # UI.message 'Bumping release version and build code...'
+    # PUBLIC_VERSION_FILE.write(
+    #   version_short: release_version_next,
+    #   version_long: build_code_code_freeze
+    # )
+    # UI.success "Done! New Release Version: #{release_version_current}. New Build Code: #{build_code_current}"
+
+    # # Bump the internal release version and build code and write it to the `xcconfig` file
+    # UI.message 'Bumping internal release version and build code...'
+    # INTERNAL_VERSION_FILE.write(
+    #   # The external and internal release versions are always the same. Because the external release version was
+    #   # already bumped, we want to just use the `release_version_current`
+    #   version_short: release_version_current,
+    #   version_long: build_code_code_freeze_internal
+    # )
+    # UI.success "Done! New Internal Release Version: #{release_version_current_internal}. New Internal Build Code: #{build_code_current_internal}"
+
+    # commit_version_and_build_files
+
+    # new_version = release_version_current
+
+    # release_notes_source_path = File.join(PROJECT_ROOT_FOLDER, 'RELEASE-NOTES.txt')
+    #extract_release_notes_for_version(
+    #  version: new_version,
+    #  release_notes_file_path: release_notes_source_path,
+    #  extracted_notes_file_path: extracted_release_notes_file_path(app: :wordpress)
+    #)
+    ## It would be good to update the action so that it can:
+    ##
+    ## - Use a custom commit message, so that we can differentiate between
+    ##   WordPress and Jetpack
+    ## - Have some sort of interactive mode, where the file is extracted and
+    ##   shown to the user and they can either confirm and let the lane commit,
+    ##   or modify it manually first and then run through the
+    ##   show-confirm-commit cycle again
+    ##
+    ## In the meantime, we can make due with a duplicated commit message and the
+    ## `print_release_notes_reminder` at the end of the lane to remember to make
+    ## updates to the files.
+    #extract_release_notes_for_version(
+    #  version: new_version,
+    #  release_notes_file_path: release_notes_source_path,
+    #  extracted_notes_file_path: extracted_release_notes_file_path(app: :jetpack)
+    #)
+    #ios_update_release_notes(
+    #  new_version:,
+    #  release_notes_file_path: release_notes_source_path
+    #)
+
+    #unless skip_user_confirmation || UI.confirm('Ready to push changes to remote to let the automation configure it on GitHub?')
+    #  UI.message("Terminating as requested. Don't forget to run the remainder of this automation manually.")
+      # next
+    # end
+
+    # push_to_git_remote(tags: false)
+
+    # attempts = 0
+    # begin
+      # attempts += 1
+      # copy_branch_protection(
+      #   repository: GITHUB_REPO,
+      #   from_branch: DEFAULT_BRANCH,
+      #   to_branch: release_branch_name,
+      #   github_token: get_required_env('GITHUB_TOKEN')
+      # )
+    # rescue StandardError => e
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/f4fe8141793a8f07d2150b6f177aaef7dbd79935/fastlane/lanes/release.rb#L116"></span>Layout/CommentIndentation: Incorrect indentation detected (column 4 instead of 6).</td>
    </tr>
  </tbody>
</table>